src/code_8006E9C0.h

```diff
@@ -23,6 +35,10 @@ extern s8  D_8018EDF3;
 extern u8 *D_8018D1E0;
 extern intptr_t D_8018D9B0;
 
+// Destination for decompressed exhaust textures (I think)
+// I'm not how much I like this "pointer to an array" setup, but if it matches it matches
+extern u8 (*D_8018D220)[1024];
```

Comment:
I could see this being like

`extern ExhaustTexture D_8018D220[num_exhause_textures];`